modules/serverless-logic/pages/cloud/build-and-deploy-with-serverless-operator-on-kubernetes.adoc

```diff
@@ -0,0 +1,193 @@
+= Building and deploying a {context} application on Kubenetes using the {product_name} Serverless Operator
+:compat-mode!:
+// Metadata:
+:description: Build and deploy using the Kogito Serverless Workflow Operator a serverless workflow application
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, openshift
+// links
+:kogito_serverless_operator_url: https://github.com/kiegroup/kogito-serverless-operator/
+:kogito_greeting_example_url: https://github.com/kiegroup/kogito-examples/tree/stable/serverless-workflow-examples/serverless-workflow-greeting-quarkus
+:kaniko_issue_url: https://github.com/GoogleContainerTools/kaniko/issues/2201
+
+This document describes how to build and deploy your workflow application using a Kubernetes cluster with the link:{kogito_serverless_operator_url}[{product_name} Serverless Operator]. If you don't have any cluster operational, link:{minikube_url}[Minikube] commands are given throughout the guide.
```

Comment:
```suggestion
[WARN]
====
This feature is meant for a quick preview on community repositories, and it's not part of the Developer Preview.
====

This document describes how to build and deploy your workflow application using a Kubernetes cluster with the link:{kogito_serverless_operator_url}[{product_name} Serverless Operator]. If you don't have any cluster operational, link:{minikube_url}[Minikube] commands are given throughout the guide.
```

@kaldesai can you please elaborate better this `WARN`?

cc @radtriste

Reply:
My suggestion would be: Instead of a `warning` can we convert it into a `note`? But the rest of the content looks good to me. 

[NOTE]
====
This feature is meant for a quick preview on community repositories, and it's not part of the developer preview.
====